cloudformation/media-atom-maker.json

```diff
@@ -55,6 +55,10 @@
             "Description": "Name of the upload tracking dynamo table",
             "Type": "String"
         },
+        "ManualPlutoTable": {
```

Comment:
I tried to merge them once but I think it we'd have to migrate the data to a new table.